lms/templates/sudo/sudo.html

```diff
@@ -1,21 +1,24 @@
 {% extends "main_django.html" %}
-{% load i18n %}
+{% load i18n third_party_auth_links %}
 
 {% block body %}
-<section style="margin: 0 auto; width: 480px; padding: 50px;">
+<section class="sudo" style="margin: 0 auto; width: 480px; padding: 50px;">
     <div class="inner-wrapper">
         <header>
           <h2 style="text-align: center;">{% trans "Confirm Your Password to Access the Instructor Dashboard" %}</h2>
         </header>
         <hr />
-        <div style="margin: 0px auto; width: 218px;">
+        <div style="margin: 0px auto; width: 320px;">
             <form class="sudo-form" method="post">{% csrf_token %}
             {{ form.as_p }}
             <p>
-                <input type="submit" value="Submit" />
+                <input type="submit" style="width: 100%;" value="{% trans 'Submit' %}" >
             </p>
             </form>
         </div>
     </div>
+
+    {% third_party_auth_links %}
+
 </section>
 {% endblock %}
```

Comment:
@dan-f We can't move this to ```dajngo_sudo_helpers``` because this is not used in ```dajngo_sudo_helpers```, its being loaded in third party module https://github.com/mattrobenolt/django-sudo. We need to place it in root of template dir like this ```templates/sudo/sudo.html```.